src/components/CommonLinks.tsx

```diff
@@ -20,7 +20,7 @@ function UpdateInstitutionProfile({
     <Link href='/update-financial-profile'>
```

Comment:
Heyo! I don't these links are working because they need to use the new url `/institution/${lei}/update`. They're currently throwing 500s when clicked.

![Screenshot 2024-03-28 at 2 14 41 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/2451f757-ac27-4142-aeef-d9402d45ff58)

You could change it like this...

https://github.com/cfpb/sbl-frontend/blob/7c12dc214155ce9bbb689dba5d427fc552e9181c/src/components/CommonLinks.tsx#L20-L22

Reply:
Great catch and very effective solution, thanks!

Reply:
Great! We'll get this reviewed by Natalia after the DSR meeting. 👍